apiml-common/src/main/java/org/zowe/apiml/product/web/HttpConfig.java

```diff
@@ -45,6 +45,8 @@ public class HttpConfig {
 
     private static final char[] KEYRING_PASSWORD = "password".toCharArray();
 
+    @Value("${server.attls.enabled:false}")
```

Comment:
Actually, we should verify both properties `server.attls.enabled` and `server.attls.required`.
```suggestion
    @Value("${server.attls.required:false} or ${server.attls.enabled:false}")
```

Reply:
Not for APIML, sorry